src/main/kotlin/inc/evil/d17map/security/authorization/policy/PolicyAuthorizationManager.kt

```diff
@@ -0,0 +1,48 @@
+package inc.evil.d17map.security.authorization.policy
+
+import org.springframework.http.HttpMethod
+import org.springframework.security.authorization.AuthorizationDecision
+import org.springframework.security.authorization.AuthorizationManager
+import org.springframework.security.core.Authentication
+import org.springframework.security.web.access.intercept.RequestAuthorizationContext
+import org.springframework.web.util.ContentCachingRequestWrapper
+import java.util.*
+import java.util.function.Supplier
+
+class PolicyAuthorizationManager : AuthorizationManager<RequestAuthorizationContext> {
+    @Deprecated("Deprecated in Java")
```

Comment:
Dlaczego używamy tutaj Deprecated?

Reply:
Z tego co się orientuję, to jeszcze Spring nie udostępnił zamiennika, ale sprawdzę jeszcze raz, bo nie pamiętam dokładnie o co z tym chodziło